api-report/map.api.md

```diff
@@ -112,6 +112,10 @@ export interface ISharedDirectoryEvents extends ISharedObjectEvents {
     (event: "valueChanged", listener: (changed: IDirectoryValueChanged, local: boolean, target: IEventThisPlaceHolder) => void): any;
     // (undocumented)
     (event: "clear", listener: (local: boolean, target: IEventThisPlaceHolder) => void): any;
+    // (undocumented)
+    (event: "subDirectoryCreated", listener: (changed: ISubDirectoryCreated, local: boolean, target: IEventThisPlaceHolder) => void): any;
+    // (undocumented)
+    (event: "subDirectoryDeleted", listener: (changed: ISubDirectoryDeleted, local: boolean, target: IEventThisPlaceHolder) => void): any;
```

Comment:
Please add TSDoc strings for the new events.

Reply:
Is there any command to add them? I think I added just like other TsDoc strings/comments in interfaces.ts?

Reply:
Yeah, they're added as comments to the source code. Syntax reference is here: <https://tsdoc.org/>